src/main/java/com/example/restservice/controller/CityController.java

```diff
@@ -0,0 +1,61 @@
+package com.example.restservice.controller;
+
+import com.example.restservice.entity.City;
+import com.example.restservice.model.CityRequestModel;
+import com.example.restservice.model.CityResponseModel;
+import com.example.restservice.model.CityResponseModelGetCities;
+import com.example.restservice.service.CityService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import java.util.List;
+
+@RestController
+public class CityController {
+
+    private CityService cityService;
+
+    @Autowired
+    public void setCityService(CityService cityService){
+        this.cityService = cityService;
+    }
+
+    @GetMapping(path = "/citiesList")
```

Comment:
in general, it looks like not well-designed REST API

it is better to have: `GET /cities` endpoint instead

Reply:
changed